### PR TITLE
net-misc/seafile-client: Bump to version 7.0.2

### PR DESCRIPTION
**Pull request by evdenis**
seafile && seafile-client versions upgrade

### CODE REVIEW

**Comment by juippis** [comment]
Would you like to become the proxy-maintainer for this package? Just follow https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

Also I see upstream has enabled python3 support, albeit I dont know if its available in the release. Could you confirm it though?

Then Im getting, 
```
!!! Fetched file: seafile-7.0.2.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      698373
!!! Expected: 698360
Refetching... File renamed to '/var/cache/distfiles/seafile-7.0.2.tar.gz._checksum_failure_.yiwszvp_'

!!! Couldn't download 'seafile-7.0.2.tar.gz'. Aborting.
 * Fetch failed for 'net-misc/seafile-7.0.2', Log file:
 *  '/tmp/portage/net-misc/seafile-7.0.2/temp/build.log'
```

File: net-misc/seafile-client/seafile-client-7.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake-utils xdg-utils
```
Comment (juippis) [inline net-misc/seafile-client/seafile-client-7.0.2.ebuild]: We usually leave one empty line below EAPI to follow skel format.

File: net-misc/seafile/seafile-7.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline net-misc/seafile/seafile-7.0.2.ebuild]: Same here. Otherwise looks good!

File: net-misc/seafile-client/seafile-client-7.0.2.ebuild (hunk 1 above)
Comment (evdenis) [reply net-misc/seafile-client/seafile-client-7.0.2.ebuild]: Done.

File: net-misc/seafile/seafile-7.0.2.ebuild (hunk 2 above)
Comment (evdenis) [reply net-misc/seafile/seafile-7.0.2.ebuild]: Done.

**Comment by evdenis** [comment]
> Would you like to become the proxy-maintainer for this package?

Record added.

> Also I see upstream has enabled python3 support, albeit I dont know if its available in the release. Could you confirm it though?

Can't see python3 support in this release. I think it will be added in the next one.

```
grep -nre 'python[23]' .
./scripts/seaf-cli-wrapper.sh:12:    if which python2.7 2>/dev/null 1>&2; then
./scripts/seaf-cli-wrapper.sh:13:        PYTHON=python2.7
./scripts/seaf-cli-wrapper.sh:14:    elif which python27 2>/dev/null 1>&2; then
./scripts/seaf-cli-wrapper.sh:15:        PYTHON=python27
./scripts/seaf-cli-wrapper.sh:16:    elif which python2.6 2>/dev/null 1>&2; then
./scripts/seaf-cli-wrapper.sh:17:        PYTHON=python2.6
./scripts/seaf-cli-wrapper.sh:18:    elif which python26 2>/dev/null 1>&2; then
./scripts/seaf-cli-wrapper.sh:19:        PYTHON=python26
./scripts/seaf-cli-wrapper.sh:46:SEAFILE_PYTHON_PATH=${INSTALLPATH}/lib/python2.6/site-packages:${INSTALLPATH}/lib64/python2.6/site-packages:${INSTALLPATH}/lib/python2.7/site-packages:${INSTALLPATH}/lib64/python2.7/site-packages
./debian/libseafile0.install:2:usr/lib/python2.7/dist-packages/seafile/*.py
./debian/rules:5:       dh $@ --with python2 --with autotools_dev
```

> Then Im getting,
```
!!! Fetched file: seafile-7.0.2.tar.gz VERIFY FAILED!
```

I've recreated manifest. Looks like the archive was silently updated.

**Comment by juippis** [comment]
Thanks, all works well!

**Pull request closed without merging**